packages/n4s/src/rules/isValueOf.ts

```diff
@@ -0,0 +1,13 @@
+import bindNot from 'bindNot';
+import { isNotNullish } from 'isNullish';
+import { isString, isPlainObject } from 'lodash';
+
+export function isValueOf(value: string, objectToCheck: object): boolean {
+  return (
+    isNotNullish(objectToCheck) &&
+    isPlainObject(objectToCheck) &&
+    isString(value) &&
+    Object.values(objectToCheck).includes(value)
+  );
```

Comment:
I think we could squeeze in a bit more perf by using a loop directly. This example is more verbose, but it should be more performant when dealing with larger datasets.

```suggestion

  if (isNullish(objectToCheck)) {
    return false;
  }

  for (const key in objectToCheck) {
    if (objectToCheck[key] === value) {
      return true;
    }
  }
  
  return false;
```